src/ROUTES.ts

```diff
@@ -945,14 +945,14 @@ const ROUTES = {
             `${action as string}/${iouType as string}/start/${transactionID}/${reportID}/distance-new/${backToReport ?? ''}` as const,
     },
     DISTANCE_REQUEST_CREATE_TAB_MAP: {
-        route: 'map/:backToReport?',
+        route: 'distance-map/:backToReport?',
```

Comment:
Why are you adding the `distance` prefix? I originally had that as well, but we decided to not use it.